### DISCUSSION TITLE
Deleting types

### DISCUSSION THREAD

**Discussion by kangaroo72** [dani-garcia/vaultwarden#3780] [Q&A]
Hi...
Is it possible to delete the pre-defined types? ![Screenshot_20230815_010034.jpg](https://github.com/dani-garcia/vaultwarden/assets/6906390/80c3b5a1-aeb0-4a82-8f1d-5ef5c07c143c)

Thx, Tom

**Comment by tessus** [comment]
Nope, you cannot.
